tools/gen-device-svd.py

```diff
@@ -439,13 +458,13 @@ def writeGo(outdir, device):
                         padNumber += 1
                         subaddress += bytesNeeded
                     if subregister['array'] is not None:
-                        subaddress += subregister['elementsize'] * subregister['array']
+                        subaddress += subregSize * subregister['array']
```

Comment:
Why not just `subregister['elementsize']`?

Reply:
I made a few changes, because it is not needed until later. 

Later we need `subregSize` because it could be either the size of the subregister element, or the size multiplied by the array size of it itself is also an array, such as the SAMD51 has in the next update for the CMSIS-SVD files.